go.mod

```diff
@@ -5,40 +5,40 @@ go 1.23.1
 replace github.com/els0r/goProbe/plugins/contrib => ./plugins/contrib
 
 require (
-	github.com/danielgtaylor/huma/v2 v2.25.0
+	github.com/danielgtaylor/huma/v2 v2.27.0
 	github.com/els0r/goProbe/plugins/contrib v0.0.0-00010101000000-000000000000
 	github.com/els0r/telemetry/logging v0.0.0-20241007081432-5f966df95bbd
 	github.com/els0r/telemetry/metrics v0.0.0-20241007081432-5f966df95bbd
 	github.com/els0r/telemetry/tracing v0.0.0-20241007081432-5f966df95bbd
 	github.com/fako1024/gotools/bitpack v0.0.0-20240726111648-469fe0cf3902
 	github.com/fako1024/gotools/concurrency v0.0.0-20240726111648-469fe0cf3902
 	github.com/fako1024/httpc v1.1.1
-	github.com/fako1024/slimcap v1.0.5
-	github.com/gin-contrib/cors v1.7.2
-	github.com/gin-contrib/pprof v1.5.0
+	github.com/fako1024/slimcap v1.0.6-0.20250102130210-2195184e7641
```

Comment:
Will be bumped to next tag once dependency PR is approved / merged: https://github.com/fako1024/slimcap/pull/88

Reply:
Bumped.